### ISSUE TITLE
M1 Support Xendit.xcframework

### ISSUE THREAD

**Issue by hendych** [xendit/xendit-sdk-ios-src#42]
When does the sdk will support m1 arch?

I know you can exclude arm64 arch and use rosetta. But from all 3rd parties that we used, this is the only 3rd party didn't support arm64 simulator

**Comment by RayYuanGH** [comment]
the same

**Comment by RayYuanGH** [comment]
"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;

**Comment by flushr** [comment]
Need this also, I'm wrapping your Android & iOS SDK for use in Flutter, no problem with Android. I think it'll be much easier to develop and debug using simulator than using real device.

**Comment by flushr** [comment]
Yes, it would be great to be able to use arm64 simulator for development and testing

**Comment by hendych** [comment]
`"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;` does not answer this issue. It will just exclude arm64 arch. What I want is to support native arm64
